db/migrate/20221202110535_create_accounts.rb

```diff
@@ -0,0 +1,12 @@
+class CreateAccounts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :accounts do |t|
+      t.string :name, nil: false
+      t.string :slug, nil: false
+
+      t.timestamps
+    end
+
+    add_reference :accounts, :user, index: true
```

Comment:
Quando vc cria essas referências, acho que é sempre bom definir o `on_delete`. Não sei qual é o valor default caso vc não coloque nada, será que é `cascade`? 🤔

Reply:
Inclusive acho que vc pode jogar essa linha aqui pra dentro do bloco lá em cima, aí a sintaxe é tipo assim:
```suggestion
    t.references :user, null: false, index: true, on_delete: :cascade
```